client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -55,9 +58,27 @@ export default React.createClass( {
 		);
 	},
 
+	renderStoreStep() {
+		switch ( abtest( 'signupStoreBenchmarking' ) ) {
+			case 'bluehost':
+			case 'bluehostWithWoo':
+				return (
+					<BluehostStoreStep { ... this.props } onBackClick={ this.handleStoreBackClick } />
+				);
```

Comment:
There's no need for the parenthesis here. We can just return it all on the same line.

```js
return <BluehostStoreStep … />;
```